src/Composer/ScriptHandler.php

```diff
@@ -43,7 +40,7 @@ public static function installAssets(CommandEvent $event, $options = false)
         $filesystem->remove($targetDir);
         $filesystem->mkdir($targetDir, $dirMode);
 
-        foreach (['css', 'fonts', 'img', 'js'] as $dir) {
+        foreach (array('css', 'fonts', 'img', 'js') as $dir) {
```

Comment:
No stress… apologies if that came out rudely, I worked though the night am was in automatic :grin: